pkg/cache/v3/delta.go

```diff
@@ -22,50 +22,27 @@ import (
 	"github.com/envoyproxy/go-control-plane/pkg/server/stream/v3"
 )
 
-// Respond to a delta watch with the provided snapshot value. If the response is nil, there has been no state change.
-func respondDelta(ctx context.Context, request *DeltaRequest, value chan DeltaResponse, state stream.StreamState, snapshot Snapshot, log log.Logger) (*RawDeltaResponse, error) {
-	resp, err := createDeltaResponse(ctx, request, state, snapshot)
-	if err != nil {
-		if log != nil {
-			log.Errorf("Error creating delta response: %v", err)
-		}
-		return nil, nil
-	}
+type resourceContainer struct {
```

Comment:
Can you format this like so? No need for the extra white space
```go
type resourceContainer struct {
	resourceMap map[string]types.Resource
	versionMap map[string]string
	systemVersion string
}
```
Also might be worth adding a doc string on this object so people understand why we created this

Reply:
done